### ISSUE TITLE
Version 3 Tag?

### ISSUE THREAD

**Issue by dereksdev** [thephpleague/omnipay-braintree#61]
Our application is relying on version `~3.0` of this package which was aliased to `dev-master`. This does not appear to be a tag that is available via composer, so we now have an issue with dependency installation. How can we install version 3 via composer?

**Comment by barryvdh** [comment]
Okay that's strange indeed, we never tagged it. Does v4 work for you?

**Comment by dereksdev** [comment]
Yes, strange indeed. I am able to install v4, however I hoped to avoid any potentially breaking changes at this time. I suppose I can pin it to a commit for the time being if there is no other option. For reference, I've included the composer.lock information below:
